pages/privacy.js

```diff
@@ -0,0 +1,254 @@
+          </ListItem>
+          <ListItem>
+          The right to object. You have the right to object to our processing of your Personal Data.
+          </ListItem>
+          <ListItem>
+          The right of restriction. You have the right to request that we restrict the processing of your personal information.
+          </ListItem>
+          <ListItem>
+          The right to data portability. You have the right to be provided with a copy of your Personal Data in a structured, machine-readable and commonly used format.
+          </ListItem>
+          <ListItem>
+          The right to withdraw consent. You also have the right to withdraw your consent at any time where Flossbank relied on your consent to process your personal information.
+          </ListItem>
+        </UnorderedList>
+        <PrivacyTextBlock>Please note that we may ask you to verify your identity before responding to such requests.</PrivacyTextBlock>
+        <PrivacyTextBlock>You have the right to complain to a Data Protection Authority about our collection and use of your Personal Data. For more information, please contact your local data protection authority in the European Economic Area (EEA).</PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>"Do Not Sell My Personal Information" Notice for California consumers under California Consumer Privacy Act (CCPA)</FAQHeading>
+        <PrivacyTextBlock>Under the CCPA, California consumers have the right to:</PrivacyTextBlock>
+        <UnorderedList>
+          <ListItem>
+          Request that a business that collects a consumer's personal data disclose the categories and specific pieces of personal data that a business has collected about consumers.
+          </ListItem>
+          <ListItem>
+          Request that a business delete any personal data about the consumer that a business has collected.
+          </ListItem>
+          <ListItem>
+          Request that a business that sells a consumer's personal data, not sell the consumer's personal data.
+          </ListItem>
+        </UnorderedList>
+        <PrivacyTextBlock>If you make a request, we have 30 days to respond to you. If you would like to exercise any of these rights, please contact us.</PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Service Providers</FAQHeading>
+        <PrivacyTextBlock>We may employ third party companies and individuals to facilitate our Service ("Service Providers"), to provide the Service on our behalf, to perform Service-related services or to assist us in analyzing how our Service is used.</PrivacyTextBlock>
+        <PrivacyTextBlock>These third parties have access to your Personal Data only to perform these tasks on our behalf and are obligated not to disclose or use it for any other purpose.</PrivacyTextBlock>
+        <PrivacyTextBlock>Analytics</PrivacyTextBlock>
```

Comment:
this and "Payments" below don't look good on the page-- maybe bold them or something.